rust/blockstore/src/types/reader.rs

```diff
@@ -131,4 +131,11 @@ impl<
             }
         }
     }
+
+    pub async fn get_all_data(&'referred_data self) -> Vec<(&'referred_data str, K, V)> {
```

Comment:
Can't we just use get_range() with unbounded ranges? This feels unnecessary to introduce?

Reply:
good suggestion. Thank you!